### PR TITLE
feat : list & navigation components

### PR DESCRIPTION
**Pull request by dipteekhd**
Added list and navigation
Review desktop UI

https://githubbox.com/dipteekhd/KHD-UI/tree/list-navigation

### CODE REVIEW

File: components/list/list.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            </li>
+            <li class="list__input m-xs-all">
+                <label for="id-2" class="center-content">
+                    <input id="id-2" type="radio" class="m-xs-r" name="I1">
+                    Item-2
+                </label>
+            </li>
+        </ul>
+
+        </div>
```
Comment (VishalPatil18) [inline components/list/list.html]: I think this is an extra `</div>`. This could be avoided.

File: components/list/list.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+            <div class="m-xs-all">
+                <h4>Heading-1</h4>
+            </div>
+            <li class="list__item list__item--active p-xs-vr p-sm-hr m-xs-all">
+                <a class="list__link" href="#">Link-1</a>
+            </li>
+            <li class="list__item p-xs-vr p-sm-hr m-xs-all">
+                <a class="list__link" href="#">Link-2</a>
+            </li>
+            <div class="m-xs-all ">
```
Comment (VishalPatil18) [inline components/list/list.html]: Extra space after `m-xs-all` could be avoided

File: components/list/list.html [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+                <h4>Heading-2</h4>
+            </div>
+            <li class="list__item p-xs-vr p-sm-hr m-xs-all">
+                <a class="list__link" href="#">Link-3</a>
+            </li>
+            <li class="list__item  p-xs-vr p-sm-hr m-xs-all">
```
Comment (VishalPatil18) [inline components/list/list.html:42]: Extra space after `list__item` could be avoided.

File: components/list/list.html [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+                <a class="list__link" href="#">Link-4</a>
+            </li>
+        </ul>
+    </section>
+
+    <!-- Radio list -->
+    <section class="m-sm-all">
+        <ul class="list">
+            <div class="m-xs-all ">
```
Comment (VishalPatil18) [inline components/list/list.html]: Extra space after `m-xs-all` could be avoided.

File: components/list/list.html [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+            </li>
+        </ul>
+
+        </div>
+    </section>
+
+    <!-- Check box list -->
+    <section class="m-sm-all">
+        <ul class="list">
+            <div class="m-xs-all ">
```
Comment (VishalPatil18) [inline components/list/list.html]: Extra space after `m-xs-all` could be avoided.

**Review by Kevin-Solomon — APPROVED** [review]

File: components/navigation/navigation.html [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+                        <a class="list__link" href="#">Link-3</a>
+                    </li>
+                    <li class="list__item  p-xs-vr p-sm-hr m-xs-all">
+                        <a class="list__link" href="#">Link-4</a>
+                    </li>
+                </ul>
+            </nav>
+        </aside>
+        <main>
+        </main>
```
Comment (VishalPatil18) [inline components/navigation/navigation.html]: You could avoid empty tags unless they have specific use. If you want to use it in the future, then you might add it later.

**Review by VishalPatil18 — APPROVED** [review]
A few suggestions were added. 
The code looks clean and works fine. Great work @dipteekhd 👍

File: components/navigation/navigation.html (hunk 6 above)
Comment (dipteekhd) [reply components/navigation/navigation.html]: Thanks for suggestion will correct it!

**Pull request merged**